common/txmgr/broadcaster.go

```diff
@@ -214,10 +209,6 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) star
 		return errors.New("Broadcaster is already started")
 	}
 	var err error
-	eb.txInsertListener, err = eb.eventBroadcaster.Subscribe(pg.ChannelInsertOnTx, "")
```

Comment:
please check if `pg.ChannelInsertOnTx` can be removed entirely from the repo

Reply:
Back when I checked this, the trigger [ChannelInsertOnTx](https://github.com/smartcontractkit/chainlink/blob/2dc0333318612b9680737804c1b47b7fa78802db/core/services/pg/channels.go#L5) was only used here, so I think it would be great if we can drop it from pg as well.